test/system/103-run.bats

```diff
@@ -19,6 +19,29 @@ teardown() {
   cd "$CURDIR" || return 1
 }
 
+@test "run: containerstart: Check container starts without issues" {
+
+  create_default_container
+
+  run $PODMAN start "$(get_system_id)-toolbox-$(get_system_version)"
+
+  CONTAINER_INITIALIZED=0
+
+  for TRIES in 1 2 3 4 5
+  do
+    run $PODMAN logs "$(get_system_id)-toolbox-$(get_system_version)"
+    CONTAINER_OUTPUT=$output
+    run grep 'Listening to file system and ticker events' <<< $output
```

Comment:
Replace `$output` with `$CONTAINER_OUTPUT`.

Reply:
Yes. Nice catch. Forgot to change there. :+1: